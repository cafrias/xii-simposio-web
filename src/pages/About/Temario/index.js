// @flow
import React from 'react'

import './Temario.css'

type TemarioProps = {
  areaName: string,
  topics: string[]
}

function Temario({areaName, topics}: TemarioProps) {
  return(
    <section className="temario__area">
      <h3>√Årea {areaName}</h3>
      <ol className="temario-list text-container">
        {
          topics.map((topic, i) => (
            <li className="temario-list__item" key={i}>
              {topic}
            </li>
          ))
        }
      </ol>
    </section>
  )
}

export default Temario
