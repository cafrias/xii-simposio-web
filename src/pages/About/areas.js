/*
Describes information for each area that the paper can be focused on.
*/
export default [
  {
    areaName: 'contable',
    topics: [
      'Información Contable. Elaboración de variables e indicadores en procesos robotizados.',
      'Rol del Profesional y cumplimiento de los requisitos de la Información Contable en el desarrollo e implementación de programas informáticos frente a las NCPA y las NIIF.',
      'Unidad de Medida. Variables a considerar en el ambiente interno y externo de las organizaciones.',
      'Contabilidad Gubernamental, la rendición de cuentas basadas en programas informáticos de gestión presupuestaria.',
      'La Información Contable de los EPEQ, efectos de la utilización de programas de autogestión.',
      'Los Usuarios de la Información Contable y el acceso digital a bases de datos.',
      'Información Contable para la Gestión en ambientes informatizados y/o robotizados.',
      'La Contabilidad de Gestión orientada al proceso de toma de decisiones en ambientes automatizados y robotizados.',
      'Técnicas contables de medición para el reconocimiento y registro de transacciones en procesos automatizados y robotizados.'
    ]
  },
  {
    areaName: 'auditoría',
    topics: [
      'El accionar del auditor en contextos automatizados y robotizados.',
      'El riesgo y el enfoque de la auditoría en contextos automatizados y robotizados.',
      'Las acciones interdisciplinarias en la auditoría en contextos automatizados y robotizados.',
      'La Auditoría y el Control Interno, en grupos económicos, adaptación a ambientes automatizados, robotizados y con complejidad asociada.',
      'La Contabilidad Gubernamental y el Control, en ambientes con tendencias a la robotización y automatización.',
      'Técnicas de Auditoría específicas para el desarrollo de auditorías en contextos automatizados y robotizados.',
      'La auditoría auxiliar de soportes de datos en la nube.',
      'Técnicas de auditoría específicas para el examen de nuevos negocios en contextos automatizados y robotizados.',
      'La Auditoría y sus acciones en contextos automatizados y robotizados.',
    ]
  },
  {
    areaName: 'investigación',
    topics: [
      'La Contabilidad y la Auditoría en ambientes automatizados y robotizados.',
      'Los nuevos tipos de relaciones económicas que surgen de la automatización y robotización, su contabilización y auditoría.',
      'Marco Normativo de la Contabilidad y Auditoría, criterios a adoptar frente a la automatización y robotización.',
      'La Contabilidad Social y Ambiental dentro de la automatización y robotización de procesos comerciales y administrativos.',
    ]
  }
]
