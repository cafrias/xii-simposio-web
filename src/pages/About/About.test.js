import React from 'react'
import { mount, shallow } from 'enzyme'

import About from './index'

it('should render without errors', () => {
  const wrapper = mount(<About/>)
  expect(wrapper).toHaveLength(1)
})

it('should match snapshot', () => {
  const wrapper = shallow(<About/>)
  expect(wrapper).toMatchSnapshot()
})
