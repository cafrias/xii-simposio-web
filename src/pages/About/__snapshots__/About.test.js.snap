// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<Section
  className="background_white about-container"
  id="about"
  name="¿Qué es el Simposio?"
>
  <div
    className="intro-container"
  >
    <div
      className="about__portrait"
    >
      <div
        className="about__pic-container"
      >
        <img
          alt="participantes del simposio"
          className="about__pic"
          src="about-pic@900.jpg"
          srcSet="
                        about-pic@1200.jpg 1200w,
                        about-pic@1700.jpg 1700w,
                        about-pic@2160.jpg 2160w,
                        about-pic@2400.jpg 2400w
                      "
        />
      </div>
      <div
        className="about__pic-container about__pic-container--back"
      >
        <img
          alt="participantes del simposio"
          className="about__pic about__pic--back"
          src="about-pic-2@900.jpg"
          srcSet="
                        about-pic-2@1200.jpg 1200w,
                        about-pic-2@1700.jpg 1700w,
                        about-pic-2@2160.jpg 2160w,
                        about-pic-2@2400.jpg 2400w
                      "
        />
      </div>
    </div>
    <article
      className="introduction mobile-pad"
    >
      <h2>
        Introducción
      </h2>
      <div
        className="text-container"
      >
        <p>
          El Simposio de Contabilidad y Auditoría del Extremos Sur, desde sus inicios en 2007, se propuso aprovechar el inspirador contexto que propone la ciudad de Ushuaia, con sus grandes atractivos naturales, para generar un espacio de debate y reflexión sobre temas de actualidad en las disciplinas que conforman el eje central del evento. De esta manera, los Profesores de la UBA, Ricardo Pahlen Acuña y Héctor Chyrikins, propusieron su creación a un grupo de docentes locales, los que decidieron otorgar todo su apoyo y brindar el máximo esfuerzo para la concreción de aquella primera edición y posterior consolidación a lo largo de los 10 años posteriores, con una frecuencia anual de encuentros durante la segunda quincena del mes de mayo.
        </p>
        <p>
          Desde aquel momento y hasta la presente edición, se han compartido brillantes conferencias, a cargo de prestigiosos profesionales y docentes del ámbito nacional e internacional, y se han presentado y expuesto un importante número de trabajos fruto de las investigaciones e indagaciones de los autores.
        </p>
        <p>
          El Simposio que se origina en la Facultad de Ciencias Económicas de la Universidad Nacional de la Patagonia San Juan Bosco, y que adoptara a partir de 2013 el Instituto de Desarrollo Económico e Innovación de la Universidad Nacional de Tierra del Fuego, Antártida e Islas del Atlántico Sur, se encuentra en el cronograma de eventos académicos de la Contabilidad y Auditoría a nivel nacional.
        </p>
        <p>
          Incrementar el alcance de los trabajos presentados en el Simposio, a través de la publicación de los mismos en formato digital, es uno de los objetivos estratégicos que se esperan concretar, junto con el incremento de la participación de estudiantes y docentes locales.
        </p>
        <p>
          Los premios que se otorgan a los mejores trabajos, constituyen un importante incentivo para generar una creciente calidad en la producción académico científica de los participantes.
        </p>
        <p>
          Es por todo esto que se invita a estudiantes, docentes e investigadores vinculados a la Contabilidad y la Auditoría y al público en general, a participar de este importante evento.
        </p>
        <footer
          className="about__more-info"
        >
          <small>
            Más información: 
             
            <a
              href="https://drive.google.com/open?id=17Z5EQhUtKSbZFGCrkhd4tDnoP6J3tFMG"
              rel="noopener noreferrer"
              target="_blank"
            >
              Aquí
            </a>
          </small>
        </footer>
      </div>
    </article>
  </div>
  <hr />
  <div
    className="about__content"
  >
    <article
      className="temario"
    >
      <h2>
        Temario
      </h2>
      <div
        className="temario-grid"
      >
        <Temario
          areaName="contable"
          key="0"
          topics={
            Array [
              Object {
                "subtemas": Array [
                  "Elaboración de variables e indicadores en procesos robotizados.",
                  "Elaboración de variables e indicadores para la Contabilidad Ambiental y Social.",
                ],
                "tema": "Información Contable:",
              },
              Object {
                "subtemas": Array [
                  "En el desarrollo e implementación de programas informáticos frente a las NCPA y las NIIF.",
                  "En el desarrollo e implementación de programas informáticos para la Contabilidad Ambiental y Social.",
                ],
                "tema": "Rol del Profesional y cumplimiento de los requisitos de la Información Contable:",
              },
              Object {
                "tema": "Unidad de Medida. Variables a considerar en el ambiente interno y externo de las organizaciones.",
              },
              Object {
                "subtemas": Array [
                  "Basada en programas informáticos de gestión presupuestaria.",
                  "Orientada a la inclusión de recursos naturales y variables sociales.",
                ],
                "tema": "Contabilidad Gubernamental, la rendición de cuentas:",
              },
              Object {
                "subtemas": Array [
                  "Efectos de la utilización de programas informáticos de autogestión.",
                  "La inclusión de la Contabilidad Ambiental y Social.",
                ],
                "tema": "La Información Contable de los EPEQ:",
              },
              Object {
                "tema": "Los Usuarios de la Información Contable y el acceso digital a bases de datos.",
              },
              Object {
                "tema": "Información Contable para la Gestión en ambientes informatizados y/o robotizados.",
              },
              Object {
                "subtemas": Array [
                  "Al proceso de toma de decisiones en ambientes automatizados y robotizados.",
                  "A la inclusión de variables ambientales y sociales en la Contabilidad.",
                ],
                "tema": "La Contabilidad de Gestión orientada:",
              },
              Object {
                "subtemas": Array [
                  "De transacciones en procesos automatizados y robotizados",
                  "De variables ambientales y sociales.",
                ],
                "tema": "Técnicas contables de medición para el reconocimiento y registro:",
              },
            ]
          }
        />
        <Temario
          areaName="auditoría"
          key="1"
          topics={
            Array [
              Object {
                "subtemas": Array [
                  "Contextos automatizados y robotizados.",
                  "Contextos de alto impacto ambiental y social.",
                ],
                "tema": "El accionar del auditor en:",
              },
              Object {
                "subtemas": Array [
                  "Contextos automatizados y robotizados.",
                  "Contextos de alto impacto ambiental y social.",
                ],
                "tema": "El riesgo y el enfoque de la auditoría en:",
              },
              Object {
                "subtemas": Array [
                  "Contextos automatizados y robotizados.",
                  "Contextos de alto impacto ambiental y social.",
                ],
                "tema": "Las acciones interdisciplinarias en la auditoría en:",
              },
              Object {
                "subtemas": Array [
                  "Ambientes automatizados, robotizados y con complejidad asociada.",
                  "Inclusión de variables de ambientales y sociales en la Contabilidad.",
                ],
                "tema": "La Auditoría y el Control Interno, en grupos económicos, adaptación a:",
              },
              Object {
                "subtemas": Array [
                  "Ambientes con tendencias a la robotización y automatización.",
                  "Ambientes de alto impacto ambiental y social.",
                ],
                "tema": "La Contabilidad Gubernamental y el Control en:",
              },
              Object {
                "subtemas": Array [
                  "Contextos automatizados y robotizados.",
                  "Contextos de alto impacto ambiental y social.",
                ],
                "tema": "Técnicas de Auditoría específicas para el desarrollo de auditorías en:",
              },
              Object {
                "tema": "La auditoría auxiliar de soportes de datos en la nube.",
              },
              Object {
                "tema": "Técnicas de auditoría específicas para el examen de nuevos negocios en contextos automatizados y robotizados.",
              },
              Object {
                "tema": "La Auditoría y sus acciones en contextos automatizados y robotizados.",
              },
            ]
          }
        />
        <Temario
          areaName="investigación"
          key="2"
          topics={
            Array [
              Object {
                "tema": "La Contabilidad y la Auditoría en ambientes automatizados y robotizados.",
              },
              Object {
                "tema": "Los nuevos tipos de relaciones económicas que surgen de la automatización y robotización, su contabilización y auditoría.",
              },
              Object {
                "tema": "Marco Normativo de la Contabilidad y Auditoría, criterios a adoptar frente a la automatización y robotización.",
              },
              Object {
                "tema": "La Contabilidad Social y Ambiental dentro de la automatización y robotización de procesos comerciales y administrativos.",
              },
            ]
          }
        />
      </div>
    </article>
    <article
      className="prizes"
    >
      <h2>
        Constitución de Premios
      </h2>
      <div
        className="text-container"
      >
        <p>
          Deseando reconocer el esfuerzo de los ponentes, y resaltar la calidad académica de las presentaciones realizadas, hemos constituido los siguientes premios:
        </p>
        <ul>
          <li>
            Premio Simposio Area Contabilidad
          </li>
          <li>
            Premio Simposio Area Auditoria
          </li>
          <li>
            Premio Profesor Hector Chyrikins (auditoria en el área gubernamental)
          </li>
          <li>
            Premio Profesor Ricardo Pahlen Acuña (contabilidad ambiental y social)
          </li>
          <li>
            Premio Señor Rector UNTDF (mejor trabajo de estudiantes)
          </li>
        </ul>
      </div>
    </article>
  </div>
</Section>
`;
